Source/UI/NewUI/PaymentForm/CardScheme.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CardScheme.swift
```

Comment:
Why do we need this file?

Reply:
It's a wrapper for `Card.Schema` in `checkout` SDK . Any merchant can import only `Frames` SDK then can use `Frames` SDK without importing `checkout` SDK. what do you think?

Reply:
Please suggest code

Reply:
https://github.com/checkout/frames-ios/pull/201/files#r922046294